### ISSUE TITLE
CMF and Sk-learn solvers and normalize=False again

### ISSUE THREAD

**Issue by sjekosch** [acoular/acoular#202]
Sk-learn normalize=False is deprecated in Version 1.4 and not used in 1.32.

**Comment by esarradj** [comment]
Uh oh, that is ugly: prior to 1.2 the default was `true`, then the default was `false` and now they remove the argument altogether. That is a very good example of how not to do backward compatibility. I have no idea how to solve this other than writing specialized Acoular code or removing support for prior 1.2 sklearn.

**Comment by sjekosch** [comment]
removing support for sk-learn version < 1.32 would also remove support for python 3.7. But that's maybe the easiest solution

**Comment by esarradj** [comment]
created branch [`202-cmf-and-sk-learn-solvers-and-normalize-false-again`](/acoular-dev/acoular/-/compare/master...202-cmf-and-sk-learn-solvers-and-normalize-false-again) to address this issue

**Comment by esarradj** [comment]
Oh no! Anaconda only has scikit-learn 1.3 while pip has already 1.4 ! There is no way to make it work with both versions.

**Comment by sjekosch** [comment]
mentioned in commit 3ae962d391273362cfd83c63258e90530737b096
